src/views/CloneRepo.vue

```diff
@@ -1,20 +1,110 @@
 <template>
   <ViewWrapper title="Clone down a repository">
-    <div>
+    <label for="repo-url-input">
+      <strong>
+        Enter url:
+      </strong>
+    </label>
+    <div class="repo-url-select">
+      <input
+        v-model="url"
+        id="repo-url-input"
+        class="repo-url-input"
+        placeholder="Enter url"
+      />
+      <button :disabled="invalidUrl" @click="clone">Clone</button>
+    </div>
+    <p v-if="invalidUrl">
+      <strong><em>
+        That repo already exists.
+      </em></strong>
+    </p>
+    <div class="folder-picker">
+      <ReposFolderPicker />
     </div>
   </ViewWrapper>
 </template>
 
 <script>
+import { mapActions, mapState } from 'pinia';
+
+import { alertsStore } from '@/stores/alerts.js';
+import { gitCloneStore } from '@/stores/gitClone.js';
+import { reposStore } from '@/stores/repos.js';
+
+import ReposFolderPicker from '@/components/appsettings/ReposFolderPicker.vue';
 import ViewWrapper from '@/views/ViewWrapper.vue';
 
 export default {
   name: 'CloneRepo',
   components: {
-    ViewWrapper
+    ViewWrapper,
+    ReposFolderPicker
+  },
+  data: function () {
+    return {
+      url: ''
+    };
+  },
+  methods: {
+    clone: async function () {
+      if (!this.url) {
+        return;
+      }
+
+      if (!this.reposFolder) {
+        return;
+      }
+
+      const repoPath = this.reposFolder + '/' + this.repoName;
```

Comment:
This works, but not on Windows. Since this project is a Cross-Platform Desktop App (XPDA), we need to write our code in cross-platform friendly ways. Instead of manually inserting an OS-specific slash, we can use Node's built-in `path` module to compose a string that will work for the current OS the code is executing on.

```suggestion
      const repoPath = path.join(this.reposFolder, this.repoName);
```
Though I think this line should be moved to it's own `destinationPath` computed property so it can be referenced by the `clone` method, as well as the validation code that ensures the destination doesn't already exist.